views/components/Maps/Maps.tsx

```diff
@@ -14,13 +14,15 @@ import 'leaflet/dist/leaflet.css';
 import "leaflet-defaulticon-compatibility";
 import "leaflet-defaulticon-compatibility/dist/leaflet-defaulticon-compatibility.css";
 import { LatLngExpression, latLng } from "leaflet";
-import { GPSCoordinate } from "@/stores/GPS/GPSTypes";
+import { GPS } from "@/stores/GPS/GPSTypes";
+import { GlobalPath } from "@/stores/GlobalPath/GlobalPathTypes";
```

Comment:
Is the type `GlobalPath` being used? Please remove if not

Reply:
done